### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by Alexintosh** [BTCP-community/zclassicjs#1]
Ahoy BTCP-community,

I was thinking it would be great to have some documentation on this library and I'd like to help with it.

I will probably use MkDocs and the Material theme, if that's okay with everyone.

**Comment by ch4ot1c** [comment]
Yes please! Please join our community Discord and I'll message ya - https://discord.gg/7FMT3J

**Comment by 11000000110000110100001101011010** [comment]
The original doc here https://github.com/ZencashOfficial/zencashjs

**Comment by Alexintosh** [comment]
Thanks, joined discord, my username there is :  Alestork#7656
What's yours @ch4ot1c ?

Thanks @johandjoz !
